src/App.js

```diff
@@ -28,15 +36,55 @@ export default class App extends Component {
        }))
        .catch(error => console.log(error))
    }
+
+  getCheckedRadios = () => {
+    let filterInputs = document.querySelectorAll('.radios')
+    for (var i = 0; i < filterInputs.length; i++) {
+      if(filterInputs[i].checked === true) {
+        this.state.checkedBoxes.push(filterInputs[i])
+      }
+    }
+  }
```

Comment:
We don't want to be doing standard vanilla JS dom access/manipulation to grab the values of the radio buttons here. Read up on Controlled Forms in React here: https://reactjs.org/docs/forms.html

There's not an exact example of radio buttons in this documentation, but the concept is the same:

* These radio buttons should probably all be in their own component, like `FilterOptions` or something.
* Each radio should have an `onChange` event that updates a value in state to reflect the filter that has been chosen
* The value of each radio button should be a value from state. 


I think you can simplify your filtering a lot to make this problem easier on yourself. Don't allow people to select 'Both' - just slim it down to either being able to select 'Male' or 'Female' and get that working first. Forget about all the other filter options for now and try getting a controlled radio button working for gender before tackling the rest of them.

Reply:
You're also frequently using the words 'checkedBoxes' and 'radios' which is really confusing. Make sure everyone is clear on the difference between checkboxes and radio buttons, because they do behave quite differently. Keep your terminology consistent.